assets/i18n/en.json

```diff
@@ -583,6 +585,8 @@
     "I18N_LEARNER_DASHBOARD_COMPLETED_TO_INCOMPLETE_COLLECTIONS": "<[numberMoved]> of the collections you completed have been moved to the 'in progress' section as new explorations have been added to them!",
     "I18N_LEARNER_DASHBOARD_CONTINUE_WHERE_YOU_LEFT_OFF_SECTION": "Continue where you left off",
     "I18N_LEARNER_DASHBOARD_CURRENT_GOALS_SECTION": "Current Goals",
+    "I18N_LEARNER_DASHBOARD_DECLINE_INVITATION_MODAL_BODY": "Are you sure you want to decline the joining request for <b><[groupName]></b> learner group?",
+    "I18N_LEARNER_DASHBOARD_DECLINE_INVITATION_MODAL_HEADER": "Decline Invitation?",
     "I18N_LEARNER_DASHBOARD_EMPTY_COLLECTION_PLAYLIST": "It looks like there aren't any collections in your 'Play Later' list. Head over to the library and build your own curated playlist!",
```

Comment:
I can't really tell what collections and explorations refer to here - @seanlip  ?

Reply:
@danapj An exploration means a lesson and a collection is a group of lessons. But this string is not part of this project, you can skip this. Thanks!